packages/driver/cypress/integration/e2e/multi-domain/multi_domain_yield_spec.ts

```diff
@@ -83,8 +83,8 @@ describe('multi-domain yields', { experimentalSessionSupport: true }, () => {
 
   it('throws if subject cannot be serialized and is accessed synchronously', (done) => {
     cy.on('fail', (err) => {
-      assertLogLength(logs, 6)
-      expect(logs[5].get('error')).to.eq(err)
+      assertLogLength(logs, 7)
```

Comment:
These tests changed because by attaching the log to the current command the log:added event is now fired for events happening in cross-origins.